index.js

```diff
@@ -157,3 +156,5 @@ exports.handler = async function handler() {
     console.log('Tweet response: ', response);
   }
 };
+
+exports.handler();
```

Comment:
I think you were testing and left this code here?

You can just call `npm start` that it will execute the script for you.

Reply:
Oh, indeed I forgot to remove that before commiting. When I try to `npm start` , there is no output (I assume it`s a problem with node and windows). For this reason, I was testing it calling the handler directly in the index file.

![2020-05-09_17h47_33](https://user-images.githubusercontent.com/18402838/81478360-2cb52f80-921d-11ea-8da7-59b49d669c00.png)